main-app/app/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
```

Comment:
Might be worth me having a full look at you project with you tomorrow afternoon, intend to do this with all of you every so often to steer a bit.